assets/javascripts/discourse/components/chat-message.js

```diff
@@ -142,6 +142,14 @@ export default Component.extend({
       });
     }
 
+    if (this.currentUser?.staff) {
```

Comment:
Perhaps it should be admin, not staff? It's very advanced.

Reply:
Current rebake is `is_staff? || @user.has_trust_level?(TrustLevel[4])` for posts this is why I went with staff, I didn't go with TL4 because chat has an existing guardian for can_edit_message which seemed appropriate for this. But I could add TL4 if you think that's necessary.

Reply:
I was just thinking of hiding things that most people won't need. But I'll leave it to your judgment! Whatever you think is best :)

Reply:
:+1: I went with exact parity with Discourse core and rebake for posts, I think this is what makes the most sense. People could find weird that they can rebake a post but not a chat message.